<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>option</class>
 <widget class="QDialog" name="option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo/logo.qrc">
    <normaloff>:/rc/Martz90-Circle-Settings.512.ico</normaloff>:/rc/Martz90-Circle-Settings.512.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="General">
     <property name="title">
      <string>General</string>
     </property>
     <widget class="QGroupBox" name="COM">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>31</y>
        <width>461</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="COMRate">
         <property name="text">
          <string>Baudrate :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="COMBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="COMBaud">
         <property name="currentText">
          <string>110</string>
         </property>
         <item>
          <property name="text">
           <string>110</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14400</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="COMLabel">
         <property name="text">
          <string>Port COM :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>461</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QComboBox" name="VitesseCombo">
         <item>
          <property name="text">
           <string>noeuds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>km/h</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>m/s</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="VitesseMesure">
         <property name="text">
          <string>Unité de Vitesse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TemperatureMesure">
         <property name="text">
          <string>Unité de Température</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="TemperatureCombo">
         <item>
          <property name="text">
           <string>Celsius</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fahrenheit</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="PressionMesure">
         <property name="text">
          <string>Unité de Pression</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="PressionCombo">
         <item>
          <property name="text">
           <string>Inches</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Millibar</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>410</y>
        <width>168</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="logo/logo.qrc"/>
 </resources>
 <connections/>
</ui>
